src/game/client/components/tclient/menus_tclient.cpp

```diff
@@ -2003,6 +2003,7 @@ void CMenus::RenderSettingsProfiles(CUIRect MainView)
 	DoButton_CheckBoxAutoVMarginAndSet(&g_Config.m_ClApplyProfileName, TCLocalize("Save/Load Name"), &g_Config.m_ClApplyProfileName, &LabelMid, LineSize);
 	DoButton_CheckBoxAutoVMarginAndSet(&g_Config.m_ClApplyProfileClan, TCLocalize("Save/Load Clan"), &g_Config.m_ClApplyProfileClan, &LabelMid, LineSize);
 	DoButton_CheckBoxAutoVMarginAndSet(&g_Config.m_ClApplyProfileFlag, TCLocalize("Save/Load Flag"), &g_Config.m_ClApplyProfileFlag, &LabelMid, LineSize);
+	DoButton_CheckBoxAutoVMarginAndSet(&g_Config.m_ClAllowEmptyValues, TCLocalize("Load Empty Values"), &g_Config.m_ClAllowEmptyValues, &LabelMid, LineSize);
```

Comment:
I don't think this is a accurate description
```suggestion
	DoButton_CheckBoxAutoVMarginAndSet(&g_Config.m_ClAllowEmptyValues, TCLocalize("Overwrite clan even if empty"), &g_Config.m_ClAllowEmptyValues, &LabelMid, LineSize);
```
This is the shortest I can think of but this wont fit in the current space so:
* Think of something shorter
* Make the label mutliline (don't use `DoButton_CheckBoxAutoVMarginAndSet`)
* Add alot of extra space (localization may take more space aswell)
You might want to push the load/save/delete/overwrite further right to fit this, localization may also make it larger so give it some space, you could also make it multiline,